datadotworld/client/_swagger/configuration.py

```diff
@@ -221,6 +226,6 @@ def to_debug_report(self):
         return "Python SDK Debug Report:\n"\
                "OS: {env}\n"\
                "Python Version: {pyversion}\n"\
-               "Version of the API: 0.1-SNAPSHOT\n"\
-               "SDK Package Version: 0.1.2".\
+               "Version of the API: 0.9.0\n"\
+               "SDK Package Version: 1.0.0".\
```

Comment:
What is this version number for? Where does it come from?

Reply:
These were also generated when the `swagger-codegen generate` command ran. I'm pretty sure the `Version of the API` is linked to the version stated in the `https://api.data.world/v0/swagger.json`